torch_frame/data/multi_nested_tensor.py

```diff
@@ -350,13 +350,34 @@ def dtype(self) -> torch.dtype:
 
     # Static methods ##########################################################
     @staticmethod
-    def stack(xs: List['MultiNestedTensor'],
-              dim: int = 0) -> 'MultiNestedTensor':
-        # TODO: To be implemented.
-        if len(xs) == 1:
-            return xs[0]
+    def cat(xs: List['MultiNestedTensor'],
+            dim: int = 0) -> 'MultiNestedTensor':
+        if dim == 0 or dim == -3:
```

Comment:
May be do something like dim = dim + MultiNestedTensor.dim if dim < 0 else dim rather than having `dim == 0 or dim == -3:`

Reply:
That'll require two lines. I think current one is simple enough.

Reply:
Addressed.